src/modules/market/Routes.tsx

```diff
@@ -13,19 +11,6 @@ export const MarketRoutesConfig: { [key: string]: RouteConfiguration } = {
   },
 };
 
-const LoadableContainer: LoadableComponent<any> = loadable(
-  async () => import('./screens/Market').then(module => module.Market),
-  {
-    fallback: <QueryLoadingAbsolute />,
-  },
-);
-
 export function MarketRoutes() {
-  return (
-    <PrivateRoute
-      path={MarketRoutesConfig.Market.path}
-      exact={true}
-      component={LoadableContainer}
-    />
-  );
+  return <Market />;
```

Comment:
I think it's not the best dicision. If you do like that, we'll lost the feature with loadable components. So market JS will be loaded with the main scripts on the main page. Which in turn slightly slows down the loading speed of the first page.

Am I right @BigBadAlien ?

Reply:
You are right. All routes must be loadable.

```
import { Route } from 'react-router-dom';
...
<Route
      path={MarketRoutesConfig.Market.path}
      exact={true}
      component={LoadableContainer}
    />
```

Should works

Reply:
@XuMinhui please consider this comment and update this part of the code.

Reply:
@XuMinhui Just to clarify, we need it for optimization purposes, every page must have its own bundle to speed up initial loading time